ec/acc/datasource_deployment_basic_test.go

```diff
@@ -55,8 +55,7 @@ func TestAccDatasourceDeployment_basic(t *testing.T) {
 					resource.TestCheckResourceAttrPair(datasourceName, "elasticsearch.0.https_endpoint_id", resourceName, "elasticsearch.0.https_endpoint_id"),
 					resource.TestCheckResourceAttrPair(datasourceName, "elasticsearch.0.version", resourceName, "elasticsearch.0.version"),
 					resource.TestCheckResourceAttrPair(datasourceName, "elasticsearch.0.topology.0.instance_configuration_id", resourceName, "elasticsearch.0.topology.0.instance_configuration_id"),
-					resource.TestCheckResourceAttrPair(datasourceName, "elasticsearch.0.topology.0.memory_per_node", resourceName, "elasticsearch.0.topology.0.memory_per_node"),
-					resource.TestCheckResourceAttrPair(datasourceName, "elasticsearch.0.topology.0.node_count_per_zone", resourceName, "elasticsearch.0.topology.0.node_count_per_zone"),
+					resource.TestCheckResourceAttrPair(datasourceName, "elasticsearch.0.topology.0.size", resourceName, "elasticsearch.0.topology.0.size"),
```

Comment:
Thanks for addressing my comments! Sorry, I forgot to mention in the previous review. Can you add checks for `size_resource` here as well as for the other resource kinds and the resource tests?